### PR TITLE
Set study length, set exit survey url

### LINKED ISSUE #55: Set Study Length
Right now the study length is set to 30 days, which I arbitrarily chose for testing purposes. Originally it was set to 7 days, but after 7 days of testing on my computer the add-on would automatically uninstall itself end as soon as I installed it.

@gregglind what's the length of this study?

### PR DESCRIPTION
**Pull request by bentoncase**
@gregglind r?

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const variationsMod = require('lib/variations');
 const forSetup = {
   name: 'Addon Recommendation Test', // unique for Telemetry
   choices: Object.keys(variationsMod.variations), // names of branches.
-  duration: 30,   // in days,
+  duration: 14,   // in days,
   surveyUrl: 'https://qsurvey.mozilla.com/s3/Shield-Study-Example-Survey',
```
Comment (gregglind) [inline index.js]: this needs changing too.

**Comment by bentoncase** [comment]
oh good catch! Do we have a survey url yet?

**Comment by bentoncase** [comment]
@Osmose study time is set, and url of survey is also set now. r?

**Comment by Osmose** [comment]
r+

**Pull request merged**